assets/docs/vasudevgupta7/models/wav2vec2-960h/1.md

```diff
@@ -0,0 +1,34 @@
+# Module vasudevgupta7/wav2vec2-960h/1
+
+This model is fine-tuned on 960h of LibriSpeech dataset for Automatic Speech Recognition. For fine-tuning, base layer was initialized from pre-trained [wav2vec2](https://tfhub.dev/vasudevgupta7/wav2vec2/1)).
+
+<!-- asset-path:  https://storage.googleapis.com/gsoc-weights/wav2vec2-960h/saved-model.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2-960h -->
```

Comment:
Is this a different *architecture* from `wav2vec2`, or a different dataset/training style? If it's the latter, it would be better to use the existing `wav2vec2` value.

Reply:
Architecture: `wav2vec2` + `LM head (dense layer)`

Training style:
This checkpoint is obtained by fine-tuning the `wav2vec2 (initialized from pre-trained weights) + LM head` model using supervised learning; while `wav2vec2` was obtained by pre-training on unlabelled dataset.

So both architecture & training style are different.

Reply:
Since the underlying model is wav2vec2, I think it's reasonable to say it is the same `wav2vec`. For example, if you fine-tuned a mobilenet model that was trained on ImageNet for some new task with a different classification layer, it would still use a `mobilenet` architecture.